react/index.ts

```diff
@@ -43,7 +43,7 @@ function addMetrics(order: OrderPlacedData) {
     'order',
     {
       order_id: order.transactionId,
-      amount: order.transactionSubtotal,
+      amount: order.transactionTotal,
```

Comment:
Reading their [docs](https://support.avantlink.com/hc/en-us/articles/203148879-Tracking-Integration):

> [ORDER_AMOUNT] – should be replaced with the dollar amount total for this order, not including tax or shipping and handling fees. Should be formatted as a decimal number without dollar sign or commas. It should be noted that the order amount must be above 0 for that sale to record as an affiliate sale.

I think the value here is right. 

`transactionSubtotal` only takes into account the value of the items in the order, while `transactionTotal` have the shipping and tax values as well.

Reply:
Can we use the subtotal considering any and all discounts? Right now the order value without considering these is being sent, resulting in the customer paying higher affiliate fees.